forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -54,9 +54,18 @@ pub enum SnapshotCommands {
         /// in default Forest data location.
         #[structopt(short, long)]
         snapshot_dir: Option<PathBuf>,
+        /// Fetch latest snapshot provided by filops servers, Default is set to Forest servers
```

Comment:
Googling `filops` gives me nothing. Let's use a different name for the official filecoin servers. Maybe something like `--host={forest|filecoin}`.

Reply:
okay